### PR TITLE
Apache release preparation

### PR DESCRIPTION
**Pull request by tuohai666**
Belong to #2009

### CODE REVIEW

**Comment by tuohai666** [comment]
@wu-sheng Thanks for you quick comment. I have added apache-rat as the release audit tool.

**Comment by wu-sheng** [comment]
What is release audit tool? Do you plan to provide something besides the documents and commands?

**Comment by wu-sheng** [comment]
Oh, you mean here, in this PR. You need to set exclusion rules for this.

**Comment by tuohai666** [comment]
> What is release audit tool? 
I didn't know Rat until you told me. So I turned to Apache Rat main page and found:
What Is Rat?

>Do you plan to provide something besides the documents and commands?

**Comment by wu-sheng** [comment]
Yes. You add the right rat plugin. But it would fail, please run locally, and find out what it says. You could take skywalking pom.xml as an example, we have it too.

File: pom.xml [hunk 1]
```diff
@@ -731,6 +748,45 @@
                 <groupId>org.apache.rat</groupId>
                 <artifactId>apache-rat-plugin</artifactId>
                 <version>${apache-rat-plugin.version}</version>
+                <configuration>
+                    <excludes>
+                        <exclude>**/target/**</exclude>
+                        <exclude>src/resources/assembly/assembly.xml</exclude>
+                        <exclude>src/resources/checkstyle.xml</exclude>
+                        <exclude>src/resources/ruleset.xml</exclude>
+                        <exclude>**/resources/META-INF/services/**</exclude>
```
Comment (wu-sheng) [inline pom.xml]: All setting files require Apache header.

File: pom.xml [hunk 2]
```diff
@@ -731,6 +748,45 @@
...
+                        <exclude>**/*.g4</exclude>
+                        <exclude>**/*.yaml</exclude>
```
Comment (wu-sheng) [inline pom.xml]: g4 and yaml are required Apache header.

**Comment by tuohai666** [comment]
Hi @wu-sheng, I have responded your comments in 3f31012ee17c97cd4e0ee99350e2583c761ad45d. Any more comments?

**Pull request merged**